<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MD1KR87C" name="automationTestCases.Chrome_Sub100_Meter_Exch_Hv_Ct" tests="1" failures="0" timestamp="14 Aug 2018 11:38:51 GMT" time="32.622" errors="1">
  <testcase name="main" time="32.622" classname="automationTestCases.Chrome_Sub100_Meter_Exch_Hv_Ct">
    <error type="java.lang.Error" message="Unresolved compilation problem: 
	INSERTID cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
INSERTID cannot be resolved to a variable

at pageObjectRepositories.CI_Objects_Call_Forward.btn_depart(CI_Objects_Call_Forward.java:97)
at webModule.CI_Methods_Call_Forward.addSuccessValues(CI_Methods_Call_Forward.java:92)
at automationTestCases.Chrome_Sub100_Meter_Exch_Hv_Ct.main(Chrome_Sub100_Meter_Exch_Hv_Ct.java:113)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- main -->
</testsuite> <!-- automationTestCases.Chrome_Sub100_Meter_Exch_Hv_Ct -->
